pytorch_lightning/callbacks/lambda_cb.py

```diff
@@ -0,0 +1,45 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
should the file be renamed to `lambda.py` to be consistent with the other callbacks?

Reply:
+1 for renaming to lambda.py and test_lambda.py